src/main/java/orm/dsl/ddl/DDLQueryBuilder.java

```diff
@@ -0,0 +1,27 @@
+package orm.dsl.ddl;
+
+import orm.SQLDialect;
+import orm.TableEntity;
+import orm.settings.JpaSettings;
+import orm.dsl.ddl.dialect.h2.H2CreateTableImpl;
+import orm.dsl.ddl.dialect.h2.H2DropTableImpl;
+import orm.settings.SnakeForPropertyNamingStrategy;
+
+public class DDLQueryBuilder {
+
+    private final JpaSettings settings;
+
+    public DDLQueryBuilder() {
+        this.settings = new JpaSettings()
+                .withDialect(SQLDialect.H2)
```

Comment:
```suggestion
                .withDialect(SQLDialect.MYSQL)
```

만약 Dialect를 변경하게 되면, 아래 코드도 수정해줘야 할텐데
같이 유동적으로 적용되는거면 좋을것 같은데 어떻게 해결해야할지 고민되긴하네요 ㅎ 🤔 
```

    public <ENTITY> CreateTableStep createTable(Class<ENTITY> entityClass) {
        return new H2CreateTableImpl<>(new TableEntity<>(entityClass, settings));
    }

    public <ENTITY> DropTableStep dropTable(Class<ENTITY> entityClass) {
        return new H2DropTableImpl<>(new TableEntity<>(entityClass, settings));
    }
```

Reply:
이부분은 다음 과제에 반영하도록 하겠습니다 :)